awx/main/access.py

```diff
@@ -320,6 +320,25 @@ def check_license(self, add_host_name=None, feature=None, check_expiration=True)
             elif "features" not in validation_info:
                 raise LicenseForbids(_("Features not found in active license."))
 
+    def check_org_host_limit(self, data, add_host_name=None):
+        validation_info = get_licenser().validate()
+        if validation_info.get('license_type', 'UNLICENSED') == 'open':
+            return
+
+        inventory = get_object_from_data('inventory', Inventory, data)
+        org = inventory.organization
+        if org is None or org.max_hosts == 0:
+            return
+
+        active_count = Host.objects.org_active_count(org.id)
+        if active_count > org.max_hosts:
+            raise PermissionDenied(_("Organization host limit of %s has been exceeded.") % org.max_hosts)
```

Comment:
Should we leak the actual host limit in this way? I'm thinking we should not give the actual value here.

Reply:
Oh, hm, I hadn't really realized that reading an organization was restricted to the admin role.  Given that, I think I agree.